### DISCUSSION TITLE
Unity收費風波

### DISCUSSION THREAD

**Discussion by sorn32** [1101-GS4719-C/Python-Community-GS4719-C#1621] [General]
Unity在9/12時，突然宣布要向用戶收取Runtime Fee費用，激起用戶的反彈和不信任感，雖然有調整收費政策，可是商業模式的轉換，特別是從免費轉向付費，依舊會對遊戲開發造成一定程度的影響，甚至會選擇轉向採用Unreal Engine等其他遊戲引擎。
雖然目前對個人版沒什麼影響，但大家會轉向Unreal Engine嗎?

資料出處:
https://www.ithome.com.tw/news/158965
https://www.blocktempo.com/unity-apologizes-for-controversial-new-pricing-scheme/

**Comment by lazywulf** [comment]
逃往 Godot 社群的 Unity 難民也頗多喔。目前也是能待在 Unity 啦，畢盡還是有許多公司與工作室仍在使用。還有如果是練習的話，我認為如果不是主要關注在學引擎怎用的話，是可以不用跳的。以自己的經驗，Unity 的架構其實讓我學到不少東西，像是一些重要的 design pattern 及經典的架構。

**Comment by v0ouo** [comment]
我覺得要轉向其他遊戲引擎還是很難欸，畢竟Unity的受眾實在是太大了，不論是開發者還是遊玩者，我自己也是玩過不少用Unity開發的遊戲，像是takumi3我到現在都還有在玩，我認為除非大家有意要一起跳槽，不然應該還是待在Unity比較好

**Comment by 1chooo** [comment]
感覺現階段還是只能待在 Unity 誒，現在遊戲社群的數量畢竟還是非常龐大，很多抱怨的人應該也只能乖乖付錢而非花更多的錢去重構，甚至去適配新的遊戲引擎，除非所有人都能取得一致的共識拒用 Unity，但這真的非常難，Unity 畢竟是很多工作室「過去」的首選！

**Comment by uwje** [comment]
雖然他要考慮成為收費軟體，但也不能阻擋他是不錯的軟體，希望他能在收益與成就他人間取得完美平衡
